test/code_block_dollar_fence-empty.md

```diff
@@ -10,5 +10,5 @@
 ```
 
 ```spaces
-  
```

Comment:
Another diff removed unnecessary trailing spaces, but I think these spaces should not be removed.